src/ducks/transactions/TransactionModal.jsx

```diff
@@ -39,34 +41,67 @@ const showComingSoon = t => {
 const Separator = () => <hr className={styles.TransactionModalSeparator} />
 
 export const TransactionModalRow = ({
+  children,
   iconLeft,
   iconRight,
   disabled = false,
-  text,
+  className,
   ...props
 }) => (
   <Media
-    className={cx(styles.TransactionModalRow, {
-      [styles['TransactionModalRow-disabled']]: disabled
-    })}
+    className={cx(
+      styles.TransactionModalRow,
+      {
+        [styles['TransactionModalRow-disabled']]: disabled
+      },
+      className
+    )}
     {...props}
   >
     <Img className={styles.TransactionModalRowIcon}>
       {Icon.isProperIcon(iconLeft) ? (
-        <Icon icon={iconLeft} width={16} color={palette.slateGrey} />
+        <Icon
+          icon={iconLeft}
+          width={16}
+          color={palette.slateGrey}
+          className={cx({
+            [styles['TransactionModalRowIcon-alignTop']]: props.align === 'top'
+          })}
+        />
       ) : (
         iconLeft
       )}
     </Img>
-    <Bd className={styles.TransactionModalRowContent}>{text}</Bd>
+    <Bd className={styles.TransactionModalRowContent}>{children}</Bd>
     {iconRight && <Img>{iconRight}</Img>}
   </Media>
 )
 
+const TransactionLabel = ({ label }) => (
+  <div className={styles.TransactionLabel}>{label}</div>
+)
+
+const TransactionInfo = ({ label, value }) => (
+  <div className={styles.TransactionInfo}>
+    <span className={styles.TransactionInfoLabel}>{label} :</span>
+    {value}
+  </div>
+)
+
+const _TransactionInfos = ({ account, date, type, t }) => (
```

Comment:
Sometimes, I go completely crazy and bypass the `translate()` to do `{ account, date, type }, { t }` (second argument to function components is the context). It works well :)